CONTRIBUTING.md

```diff
@@ -115,3 +115,71 @@ npm run api-gen
 ```
 
 Configuration for the docs generator is located in the `api-gen` directory.
+
+## Documentation Style Checking
+
+The Konflux documentation uses Vale for style checking to ensure consistent voice and terminology across all documentation. Vale helps maintain our documentation standards and writing style guidelines.
+
+### Prerequisites
+
+In addition to the AsciiDoc tools mentioned above, you'll need:
+
+1. **Vale** - Style checker for documentation
+   - macOS: `brew install vale`
+   - Linux: See [Vale installation guide](https://vale.sh/docs/vale-cli/installation/)
+
+The repository includes Vale configuration and style rules in the `.vale` directory.
+
+### Running Style Checks
+
+We provide a wrapper script that ensures all dependencies are installed and Vale is properly initialized. You can run style checks using npm:
+
+We enforce consistent voice and style across all documentation using Vale. Our Vale configuration enforces different levels of style rules:
+
+#### Error-level Rules (Must Fix)
+- **Direct Address**: Always address the reader as "you" rather than "the user"
+- **Imperative Mood**: Use direct commands in procedures ("Create a file" instead of "You should create a file")
+
+#### Warning-level Rules (Should Fix)
+- **Active Voice**: Prefer active voice over passive voice where possible
+- **Consistent Tense**: Use present tense by default, especially in procedures
+
+The CI pipeline will fail on error-level violations but only warn about warning-level violations. This balance ensures critical voice consistency while maintaining flexibility where needed.
+
+To check your documentation against these rules:
+```bash
+# Show all issues (warnings and errors)
+npm run lint:docs
+
+# Check all documentation
+npm run lint:docs
```

Comment:
twice the same example?

Reply:
Thanks for catching this. It was a copy/paste issue. I will resolve it on the next push.